src/style.css

```diff
@@ -1,3 +1,88 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.navigation {
+  max-height: 60px;
+  box-sizing: border-box;
+  flex-shrink: 0;
+  margin-right: 0;
+  padding: 0 50px;
+  align-items: center;
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+}
+
+.navigation-image img {
```

Comment:
Please don't style html tags. You can create class for this element and style by class.